configs/pontoon.toml

```diff
@@ -145,6 +145,13 @@ locales = [
         "pt-BR",
         "ru"
     ]
+[[paths]]
+    reference = "en/firefox/privacy/book.ftl"
+    l10n = "{locale}/firefox/privacy/book.ftl"
+    locales = [
+        "de",
+        "fr"
+    ]
```

Comment:
@alexgibson does this config look correct to you? @peiying2 & I were having issues with it being populated for all locales.

Reply:
This looks ok to me. @peiying2 we have several other ftl files above with configs for only a subset of locales. Do we have a problamne with those also, or just this new one?

Reply:
Actually this is an issue for pages offered to only select locales.

Reply:
Sounds like this is something that needs fixing on the Pontoon side then?

Reply:
How did we know it worked before?  I can ask Matjaz to check on Pontoon side.

Reply:
I think we need to check in pontoon if the other fluent files in the config are only exposed to the select locales or everyone.

Reply:
@mathjazz Please review the toml file for book.ftl page. 

In addition to this new page, two other configuration pages didn't take effect and need to be fixed at some point:
- en/mozorg/newsletters.ftl
- en/newsletter/opt-out-confirmation.ftl

Reply:
I believe the [last path definition](https://github.com/mozilla-l10n/www-l10n/blob/bf225c9f76a7c383f33b5dc0632e6f459e7d6a0f/configs/pontoon.toml#L155-L157) invalidates all the previous ones. It enables all .ftl files for all locales, so previous definitions don't have any effect.

Reply:
Oh interesting, thanks! Ok, I'll make a change to reorder upstream and we'll see if this fixes things. I'll also make sure we document this.

Reply:
The order itself isn’t significant AFAICT. You’ll need to add more specific paths I’m afraid.

Reply:
PR to fix the ordering: https://github.com/mozilla/bedrock/pull/9734

Reply:
> The order itself isn’t significant AFAICT. You’ll need to add more specific paths I’m afraid.

Oh ok, we'll need to figure this one out another way

Reply:
@mathjazz I'm reading through https://moz-l10n-config.readthedocs.io/en/latest/fileformat.html#paths, but I'm struggling to find something that gives us the flexibility to define specific .ftl files for only a subset of locales that doesn't require making definitions for just about every file in the repo.

Is there something you can help suggest here, that might not be as long winded?

Reply:
I'm afraid I don't have great ideas. :/

1. Instead of adding definitions for each file, you could add definitions for subdirectories.
2. You could put all files localized by all locales into a separate high-level directory.

Maybe @flodolo has better ideas?

Reply:
I'll see what I can work out, thanks

Reply:
I think for now, adding definitions for sub directories seems like the quickest / simplest fix. Moving all the localized files would take some level of effort / coordination, which may be worth considering in the future.

PR here: https://github.com/mozilla/bedrock/pull/9735

Reply:
> Maybe @flodolo has better ideas?

Unfortunately I haven't :-(